extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/NettyHttpClientConfig.java

```diff
@@ -111,6 +125,28 @@
         public Optional<String> threadNamePrefix;
     }
 
+    @ConfigGroup
+    public static class NettyProxyConfiguration {
+
+        /**
+         * Enable HTTP proxy
+         */
+        @ConfigItem(defaultValue = "false")
```

Comment:
`defaultValue` is not needed, boolean default to `false`

Reply:
I left it to let the doc generator to show default value